README.md

```diff
@@ -1,7 +1,7 @@
 # Calendar
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://desevoker.github.io/layout_calendar/)
+- [TEST REPORT LINK](https://desevoker.github.io/layout_calendar/report/html_report/)
```

Comment:
Are there automated tests for this task?

Reply:
Unfortunately, there aren't any.